apps/web/modules/videos/cal-video-premium-features.tsx

```diff
@@ -119,64 +68,54 @@ export const CalAiTranscribe = ({
     }, [])
   );
 
-  useDailyEvent("joined-meeting", (ev) => {
-    if (enableAutomaticTranscription) {
+  useDailyEvent("joined-meeting", () => {
+    if (enableAutomaticTranscription && !transcription?.isTranscribing) {
       daily?.startTranscription();
-      updateCustomTrayButtons({
-        recording: BUTTONS.START_RECORDING,
-        transcription: transcription?.isTranscribing
-          ? BUTTONS.STOP_TRANSCRIPTION
-          : BUTTONS.START_TRANSCRIPTION,
+    }
+    if (enableAutomaticRecordingForOrganizer && !recording?.isRecording) {
+      daily?.startRecording({
+        // 480p
+        videoBitrate: 2000,
```

Comment:
It is duplicated now, maybe let's do it like this

calvideo.startRecording _> whichj calls -> daily.startRecording with bitrate